src/Common/REST/TEC/V1/Abstracts/Post_Entity_Endpoint.php

```diff
@@ -169,7 +169,7 @@ public function get_post_type_object(): WP_Post_Type {
 	 *
 	 * @return array
 	 */
-	protected function format_post_entity_collection( array $posts ): array {
+	protected function format_entity_collection( array $posts ): array {
```

Comment:
Do we need to account for the old method name?

Also, we should update the `@since` tag to flag that the method was renamed.

Reply:
No, anything in the new REST API we declare that backwards compatibility is not a guarantee.